app/Http/Controllers/AuthController.php

```diff
@@ -21,32 +27,25 @@ class AuthController extends Controller
 {
     use TokenTrait;
 
-    public function login(LoginRequest $request, UserService $service, JWTAuth $auth): JsonResponse
+    public function login(LoginRequest $request, UserService $service, JWTAuth $auth): LoginResourceFail | LoginResourceSuccess
     {
         $credentials = $request->only('email', 'password');
         $remember = $request->input('remember', false);
 
         $token = $auth->attempt($credentials);
 
         if ($token === false) {
-            return response()->json(['message' => 'Authorization failed'], Response::HTTP_UNAUTHORIZED);
+            return LoginResourceFail::make([]);
```

Comment:
could we just throw `UnauthorizedHttpException` here without the separate resource? Please check that response structure will be the same in this case